gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -65,3 +67,55 @@ internal fun GravatarQuickEditorPage(
         }
     }
 }
+
+/**
+ * Raw composable component for the Quick Editor.
+ * This can be used to show the Quick Editor in Activity, Fragment or BottomSheet.
+ *
+ * @param gravatarQuickEditorParams The Quick Editor parameters.
+ * @param authToken The authentication token.
+ * @param onAvatarSelected The callback for the avatar update result, check [AvatarUpdateResult].
+ *                       Can be invoked multiple times while the Quick Editor is open
+ * @param onDismiss The callback for the dismiss action.
+ *                  [GravatarQuickEditorError] will be non-null if the dismiss was caused by an error.
+ */
+@Composable
+internal fun GravatarQuickEditorPage(
+    gravatarQuickEditorParams: GravatarQuickEditorParams,
+    authToken: String,
+    onAvatarSelected: (AvatarUpdateResult) -> Unit,
+    onDismiss: (dismissReason: GravatarQuickEditorDismissReason) -> Unit = {},
+) {
+    val navController = rememberNavController()
+
+    DisposableEffect(authToken) {
+        QuickEditorContainer.getInstance().useInMemoryTokenStorage()
+
+        onDispose {
+            QuickEditorContainer.getInstance().resetUseInMemoryTokenStorage()
+        }
+    }
+
+    NavHost(
+        navController,
+        startDestination = QuickEditorPage.SPLASH.name,
+        enterTransition = { EnterTransition.None },
+        exitTransition = { ExitTransition.None },
+    ) {
+        composable(QuickEditorPage.SPLASH.name) {
+            SplashPage(
+                email = gravatarQuickEditorParams.email,
+                token = authToken,
+            ) {
+                navController.navigate(QuickEditorPage.EDITOR.name)
+            }
+        }
```

Comment:
I was hoping to skip the `SplashPage` completely but there's a timing issue. 

`DisposableEffect` is run after the composition is successful, so if we only have the `AvatarPicker` its ViewModel will be initialized with the wrong dependencies, first, and after that the effect code will be run.